apiary.apib

```diff
@@ -2474,6 +2474,15 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + hourly_rate (object, required)
+            + One Of
+                + Properties
+                    + type: `user` (string, required)
+                + Properties
+                    + type: `task` (string, required)
+                + Properties
+                    + type: `fixed` (string, required)
+                    + rate: `123` (number, required)
```

Comment:
B : a rate can never be just a number... it should probably be Money (amount & currency)